MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -13,6 +13,8 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		didSet { tableView.reloadData() }
 	}
 
+	@IBOutlet weak var errorView: ErrorView!
```

Comment:
Same here for as for the MVP solution:
```suggestion
	@IBOutlet private(set) var errorView: ErrorView!
```